mteb/cmd.py

```diff
@@ -84,12 +85,6 @@ def main():
     parser.add_argument(
         "--device", type=int, default=None, help="Device to use for computation"
     )
-    parser.add_argument(
-        "--batch_size", type=int, default=32, help="Batch size for computation"
-    )
```

Comment:
I think this one is important, it's supposed to be used by the encode function of Evaluators

Reply:
But atm I don't think it is used so I chose to remove it to avoid users thinking that it is used.